### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by andrayantelo**

### CODE REVIEW

File: docs/checkit.js [hunk 1]
```diff
@@ -351,7 +351,7 @@ CheckIt.prototype.generateEmptyCalendar = function(calObj) {
         checkitApp.$calendarDiv.append('<div class="monthframe" id=' + monthObj.monthId + '></div>');
         
         if (monthObj.monthIndex === 0 || index === 0) {
-            var yearHeader = "<div class='page-header text-center' id='yearHeader'>" +
+            var yearHeader = "<div class='page-header text-center yearHeader'>" +
```
Comment (johntyree) [inline docs/checkit.js]: `yearHeader` probably should have been spelled `year-header` based on the others here

File: tests/tests.js [hunk 2]
```diff
@@ -188,7 +188,7 @@ QUnit.test("generateEmptyCalendar test", function( assert ) {
     this.checkit.addMonth(this.calendar);
     $calendarDiv.empty();
     this.checkit.generateEmptyCalendar(this.calendar, $calendarDiv);
-    assert.notOk($calendarDiv.find('20172 > #yearHeader').text(),
+    assert.notOk($calendarDiv.find('20172 > .yearHeader').text(),
     "Make sure second month does not have a yearHeader");
```
Comment (johntyree) [inline tests/tests.js]: Thought this was a syntax error. This line should be indented.

File: tests/tests.js [hunk 3]
```diff
@@ -770,11 +771,15 @@ QUnit.module( "CheckIt tests for functions that involve store", {
     localStorage.setItem(this.worldState.uniqueId, JSON.stringify(this.worldState));
     //add loading wheel
     this.$calendarDiv.append(`<div id="loadingWheel"></div>`);
+    // make calendar object out of hello calendar and world calendar
```
Comment (johntyree) [inline tests/tests.js]: Not sure this comment really adds anything. Also it should be plural.

File: tests/tests.js [hunk 4]
```diff
@@ -770,11 +771,15 @@ QUnit.module( "CheckIt tests for functions that involve store", {
...
+    this.helloCal = new Calendar(this.helloState);
+    this.worldCal = new Calendar(this.worldState);
     
   },
   afterEach: function() {
-    // clean up calendar html
+    // clean up calendar html, and reset checkit's calendar div
     this.$calendarDiv.empty()
+    this.checkit.setCalendarDiv(this.$calendarDiv);
```
Comment (johntyree) [inline tests/tests.js]: Doesn't every test get a brand new checkit object?

**Review by johntyree — CHANGES_REQUESTED** [review]

File: tests/tests.js (hunk 4 above)
Comment (andrayantelo) [reply tests/tests.js]: Yeah, you're right lol

**Review by johntyree — APPROVED** [review]

**Pull request merged**